### PR TITLE
Download external libyuv-project using DOWNLOAD_URL option

### PR DESCRIPTION
**Pull request by TBSniller**
Download of libyuv tar really often fails during first project build phase, because used curl for source.tar download doesn't seem to support https.
This PR works around this problem using the `DOWNLOAD_COMMAND` option. 

@tuxuser Don't know if this is the straight way to fix this issue. Maybe you can leave a comment.

### CODE REVIEW

**Comment by tuxuser** [comment]
Could you check how the following snippet performs?

For me it takes slightly longer to do the clone, because it has to pull the history all together; `GIT_SHALLOW false` does not work, cause its not a tag as reference but a commit hash.

Unfortunately there is no real alternative to pinning on a commit hash, as libyuv does not have any tags defined (@Informatic noted that previously). Pseudo-pinning to a branch, even if called `stable`, would not result in a reproducible build and probably introduce an additional/unnecessary point of failure in the future.

TL;DR The git way works on my machine? Are we good with that tradeoff?

```cmake
ExternalProject_Add(
  libyuv
  GIT_REPOSITORY https://chromium.googlesource.com/libyuv/libyuv
  GIT_TAG fdc71956bdade0012b0628b5011b567c06c72308
  CMAKE_ARGS
    -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
    -DCMAKE_C_FLAGS=${COMMON_FLAGS}
    -DCMAKE_CXX_FLAGS=${COMMON_FLAGS}
    -DBUILD_SHARED_LIBS=OFF
    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TOOLCHAIN_FILE}
    -DCMAKE_INSTALL_PREFIX=${CMAKE_CURRENT_SOURCE_DIR}
    -DCMAKE_POSITION_INDEPENDENT_CODE=ON
)
```

**Comment by TBSniller** [comment]
It's even quite faster, thanks for your suggestion!
